### ISSUE TITLE
Could not open the c3d file: iostream error

### ISSUE THREAD

**Issue by rmndrs89** [pyomeca/pyomeca#137]
Hi all,

thanks for this amazing work, it looks fantastic and we are now trying to incorporate (parts of) it in our analysis pipeline.

When I was going through the [Getting Started: Object Creation](https://pyomeca.github.io/getting-started/#object-creation), I encountered an `OSError`:
```python
OSError: Could not open the c3d file: iostream error
```

Steps to reproduce:
```bash
conda install -c conda-forge pyomeca
```

```python
from pyomeca import Markers

data_path = "../tests/data/markers_analogs.c3d"
markers = Markers.from_c3d(data_path, prefix_delimiter=":")
```

I am working on Ubuntu 22.04 LTS.

Any idea what does wrong here?

Best regards,
Robbin

**Comment by pariterre** [comment]
Dear @rmndrs89 

The `OSError: Could not open the c3d file: iostream error` means that the file is not found. You probably had an error in your file path :)

**Comment by rmndrs89** [comment]
Thanks @pariterre! You were right, probably some slash misplaced in the `pathlib.Path`

**Comment by pariterre** [comment]
Fixed in #322
